core/arch/arm/mm/core_mmu.c

```diff
@@ -101,6 +101,10 @@ static struct memaccess_area nsec_shared[] = {
 	MEMACCESS_AREA(CFG_SHMEM_START, CFG_SHMEM_SIZE),
 };
 
+#ifdef CFG_TEE_SDP_MEM_BASE
```

Comment:
Since I added this comment, you've rebased this, do we want to use the `CFG_SECURE_DATA_PATH` here?